rules/kube_apiserver-availability.libsonnet

```diff
@@ -59,31 +59,27 @@
               1 - (
                 (
                   # write too slow
-                  sum by (%(clusterLabel)s) (cluster_verb_scope:apiserver_request_sli_duration_seconds_count:increase%(SLODays)s{%(kubeApiserverWriteSelector)s})
+                  sum by (%(clusterLabel)s) (cluster_verb_scope:apiserver_request_sli_duration_seconds_count:increase%(SLODays)s{%(kubeApiserverWriteSelector)s} or vector(0))
```

Comment:
without this data, it is impossible to calculate a meaningful value.

This is the total number of requests the API server has received. In case no request has been received, or the data is missing, we cannot assess the availability in any statistically meaningful way.